lib/ui/components/cv_header.dart

```diff
@@ -18,17 +18,16 @@ class CVHeader extends StatelessWidget {
           title,
           style: Theme.of(context).textTheme.headline3.copyWith(
                 fontWeight: FontWeight.w400,
-                color: AppTheme.primaryColorDark,
+                color: PrimaryAppTheme.primaryColorDark,
               ),
           textAlign: TextAlign.center,
         ),
         subtitle != null
             ? Text(
                 subtitle,
                 style: Theme.of(context).textTheme.subtitle1.copyWith(
-                      fontWeight: FontWeight.bold,
-                      color: Colors.black,
-                    ),
+                    fontWeight: FontWeight.bold,
+                    color: PrimaryAppTheme.getTextColor(context)),
```

Comment:
Please add `,` after `(context)`and before `)` and run `flutter format` on it. It will result in better formatting's.